### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by eak000**
Used flexbox as easy footer fix. I know older browsers don't support though. I'll keep trying to figure than out. First I want to make the footer a little more responsive. But flexbox works awesome for newer browsers.

### CODE REVIEW

**Comment by natsteinmetz** [comment]
@eak000 Looks like it's not supported by IE 8 & 9 mostly - looking at our analytics, we have less than 2% of users using IE, and of those less than 10% using IE 9, none IE 8. I think we can live with that. :)

File: app/assets/stylesheets/blog.css.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+//
```
Comment (natsteinmetz) [inline app/assets/stylesheets/blog.css.scss]: I see that this PR is dependent on the blog related one. This is something that is great to note in the PR description, and helps prevent the too-early merging of dependent branches.

**Comment by eak000** [comment]
I didn't mean for it to be...can you explain more what that means? And how I can fix it?

**Comment by natsteinmetz** [comment]
Hm, I don't see any changes on the footer, when checking out the branch, and reducing the window size, the footer gets still the same wonkyness than on the deployed site.

**Comment by eak000** [comment]
The footer isn't terribly responsive. But the flex box solves the problem of the footer floating up on pages with less content.

**Comment by eak000** [comment]
I want to work a little on the footer and header for responsiveness. They both get wonky on my mac at smaller screen sizes

**Comment by eak000** [comment]
the changes that affect the footer are actually on .content and .page-content on the application.css

**Comment by natsteinmetz** [comment]
Generally, when you create a new branch, you create it off your master (meaning master is checked out when you create your new branch). In this case, when you created the footer branch, you were not on master, but on your 'blog' branch. Which means you created the footer related changes on top of the blog branch related changes. So this branch now contains bot the 'blog' and 'footer' branch changes.

Sometimes this is wanted behaviour, if one creates a branch that is actually dependent on work done in another branch. In this case, that is not necessary, but also not a big problem. In general this can lead to more merge conflicts, if lots of changes are happening on the underlying branch ('blog' in our case). So after you have done the change on the blog branch (the top-margin), you should probably merge that branch into this one.

**Comment by eak000** [comment]
Thanks for the explanation! I remember thinking I needed to switch to master, but I must have not done so. Whoops!

**Comment by natsteinmetz** [comment]
@eak000 I know it's been ages (apologies!!!), but is there any chance you'd like to look at those merge conflicts? I would merge it in once those are resolved. I can also take care of it myself, just wanted to check in with you. :)

**Comment by eak000** [comment]
Natalie,
Wow, it has been ages. I haven't used Git in ages (possibly since this was originally written), and merge conficts always made my palms sweaty anyway....can you fix it? I have gone down another path at the moment...I'm opening a makerspace!

**Comment by bambery** [comment]
@natsteinmetz - what was the original task, here? I can take a look at it.

**Comment by jadefaerie** [comment]
@eak000 Hey Emily! I just wanted to thank you for your help and let you know that we're doing a complete overhaul of the site, so we will be closing out this PR. I hope things are going well with the makerspace -- or whatever you're doing these days :) If you get back into coding, let us know! Good luck to you!

**Pull request closed without merging**